crates/store/re_protos/proto/rerun/v1alpha1/jobs.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package rerun.redap_jobs.v1alpha1;
+
+import "rerun/v1alpha1/common.proto";
+
+// JobsService is the service for submitting and querying persistend redap tasks.
+service JobsService {
```

Comment:
It seems like the notion of a job doesn't exist at all anymore, should this be a `TasksService`?